docs/output.md

```diff
@@ -58,19 +58,29 @@ Most plots are included in the HTML report (see above), but are also included in
 <summary>Output files</summary>
 
 - `tables/`
+
   - `annotation1/`: Directory containing annotation matrices generated in the course of analysis
     - `[array platform].annotation.tsv`: Annotations derived from an array platform
     - `[GTF name].anno.tsv`: Species wise annotations derived from a GTF in RNA-seq analysis
   - `processed_abundance/`: Directory containing processed abundance values from initial processing from e.g. DESeq2 or Affy:
-    - `[contrast_name].normalised_counts.tsv`: Normalised counts table (DESeq2)
+    - `[contrast_name].normalised_counts.tsv`: Normalized counts table (DESeq2). Rows are ensembl IDs and columns are samples.
     - `[contrast_name].vst.tsv`: Normalised counts table with a variance-stabilising transform (DESeq2)
     - `raw.matrix.tsv`: RMA background corrected matrix (Affy)
     - `normalised.matrix.tsv`: RMA background corrected and normalised intensities matrix (Affy)
   - `differential/`: Directory containing tables of differential statistics reported by differential modules such as DESeq2
+
+    - `[contrast_name].deseq2.results.tsv`: Results of DESeq2 differential analysis (RNA-seq)
+    - `OR [contrast_name].limma.results.tsv`: Results of Limma differential analysis (Affymetrix arrays)
+
+  - `gsea/`: Directory containing tables of differential gene set analysis from GSEA (where enabled)
+
     - `[contrast_name].[deseq2|limma].results.tsv`: Results of DESeq2 differential analyis (RNA-seq) OR Limma differential analysis (Affymetrix arrays, GEO studies, Maxquant proteomics studies)
     - `[contrast_name].[deseq2|limma].results_filtered.tsv`: Results of DESeq2 differential analyis (RNA-seq) OR Limma differential analysis (Affymetrix arrays, GEO studies, Maxquant proteomics studies); filtered for differentially abundant entries
```

Comment:
```suggestion
    - `OR [contrast_name].limma.results.tsv`: Results of limma differential analysis (Affymetrix arrays)

  - `gsea/`: Directory containing tables of differential gene set analysis from GSEA (where enabled)

    - `[contrast_name].[deseq2|limma].results.tsv`: Results of DESeq2 differential analyis (RNA-seq) OR limma differential analysis (Affymetrix arrays, GEO studies, Maxquant proteomics studies)
    - `[contrast_name].[deseq2|limma].results_filtered.tsv`: Results of DESeq2 differential analyis (RNA-seq) OR limma differential analysis (Affymetrix arrays, GEO studies, Maxquant proteomics studies); filtered for differentially abundant entries
```

The _limma_ authors published the tool in lowercase (see [publication](https://academic.oup.com/nar/article/43/7/e47/2414268)).